community/bolt/transport-socket/pom.xml

```diff
@@ -50,6 +50,15 @@
     </dependency>
 
     <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
Do we need to add it to Bolt in 2.3? Since we dislodged Bolt from the deps, we shouldn't need it right? Suggest we do a separate PR for adding it to Bolt in 3.0, since the modules have changed there. (in fact, I've got a card to remove Bolt entirely from the 2.3 branch now that 2.3.0 is out).